src/test/java/com/datadoghq/sketch/ddsketch/DDSketchTest.java

```diff
@@ -154,7 +154,7 @@ protected void test(boolean merged, double[] values, DDSketch sketch) {
     }
 
     void testProtoRoundTrip(boolean merged, double[] values, DDSketch sketch) {
-        assertEncodes(merged, values, DDSketch.fromProto(storeSupplier(), sketch.toProto()));
+        assertEncodes(merged, values, DDSketchProtoBinding.fromProto(storeSupplier(), DDSketchProtoBinding.toProto(sketch)));
```

Comment:
Can we also test here serializing with `DDSketch.serialize()` and deserializing with `DDSketchProtoBinding .fromProto`, and possibly, the equality of the serialized byte arrays produced with `DDSketch.serialize` and `DDSketchProtoBinding.toProto`? We can also rename the method.

Same comment for `StoreTest.testProtoRoundtrip` and `LogLikeIndexMapping.testProtoRoundTrip`.